### PR TITLE
Add a console command user:add to create users over the console

### PR DESCRIPTION
**Pull request by nickvergessen**
To test put 

``` sh
COUNTER=1
while [  $COUNTER -lt 1000 ]; do
    ./occ user:add test-$COUNTER -p "testtest" -g "stressTest" --display-name "Test user $COUNTER"
    let COUNTER=COUNTER+1 
done
```

Into a .sh and flood it with test users ;)
Used it to create some dummies for performance tests.

@DeepDiver1975 @LukasReschke @MorrisJobke

### CODE REVIEW

**Comment by MorrisJobke** [comment]
Works :+1:

Awesome :)

@jospoortvliet Something for your dev report ;)

**Comment by MorrisJobke** [comment]
And of course: the code looks good :)

File: core/command/user/add.php [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+class Add extends Command {
+	/** @var \OC\User\Manager */
+	protected $userManager;
+
+	/** @var \OC\Group\Manager */
+	protected $groupManager;
+
+	/**
+	 * @param \OC\User\Manager $userManager
```
Comment (LukasReschke) [inline core/command/user/add.php]: :see_no_evil:

Comment (MorrisJobke) [inline core/command/user/add.php]: Maybe also use `\OCP\IUserManager` and `\OCP\IGroupManager` here.

**Comment by karlitschek** [comment]
very cool! :+1:

File: core/command/user/add.php (hunk 1 above)
Comment (DeepDiver1975) [inline core/command/user/add.php]: ~~Maybe~~ mandatory! - THX

Comment (nickvergessen) [inline core/command/user/add.php]: My laptop died right before lunch, fix is on the way

Comment (nickvergessen) [inline core/command/user/add.php]: done

**Comment by MorrisJobke** [comment]
Still works :+1:

**Pull request merged**